logging/src/main/java/org/jboss/as/logging/logging/LoggingLogger.java

```diff
@@ -840,7 +840,7 @@
      * @return an {@link OperationFailedException} for the error
      */
     @Message(id = 79, value = "Failed to read the log file '%s'")
-    RuntimeException failedToReadLogFile(@Cause Throwable cause, String name);
+    OperationFailedException failedToReadLogFile(@Cause Throwable cause, String name);
```

Comment:
Maybe that's why I had it a `RuntimeException` to begin with. I was kind of kicking myself for not making a note of why I changed that in the first place to a `RuntimeException`.